src/qt/numeric-formats/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+set(MODULE_NAME numeric-formats)
```

Comment:
This raises a good question about where to put Qt-related modules. Personally I would just put in under `modules`, but with `qt-` as prefix

Reply:
I still feel that this library should be under `libraries`, not under `src`.

Probably under `libraries/qt`.

Reply:
> This raises a good question about where to put Qt-related modules. Personally I would just put in under `modules`, but with `qt-` as prefix

Note that I have begin using a `lib-wx-` naming convention.